graylog2-web-interface/src/theme/GlobalThemeStyles.ts

```diff
@@ -152,6 +152,10 @@ const GlobalThemeStyles = createGlobalStyle(
       margin-left: 5px;
     }
 
+    button {
+      line-height: ${theme.fonts.lineHeight.body};
+    }
```

Comment:
This change is fixing a small styling issue with dropdowns in the navigation. Before their line height was slightly different compared with navigation links. This change does not affect the `Button` component, but in my opinion it would look better if we also apply this line-height for `Button`s.

Before:
![image](https://github.com/user-attachments/assets/7b05fabd-0822-4068-b052-fcf8f1789973)

After:
![image](https://github.com/user-attachments/assets/082d30a8-f3c9-451e-8919-b1c860c2b9e4)